zerver/lib/home.py

```diff
@@ -153,6 +153,7 @@ def build_page_params_for_home_page_load(
         "user_settings_object": True,
         "linkifier_url_template": True,
         "user_list_incomplete": True,
+        "archived_streams": True,
```

Comment:
I think in this first commit, where the web app doesn't actually support seeing archived streams, this should probably be `False`? That way, this commit works.

Reply:
Yeah makes sense. Fixed it.